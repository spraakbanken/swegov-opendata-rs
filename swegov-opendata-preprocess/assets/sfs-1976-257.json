{"dokumentstatus":{"dokument":{"dok_id":"sfs-1976-257","hangar_id":"","rm":"1976","beteckning":"1976:257","typ":"sfs","subtyp":"sfst","dokumentnamn":"Svensk författningssamling","tempbeteckning":"","organ":"Bostadsdepartementet","nummer":"257","slutnummer":"0","datum":"1976-05-13 00:00:00","publicerad":"2016-02-11 15:25:48","systemdatum":"2016-02-11 15:25:48","titel":"Förordning (1976:257)\r\nom lån för förvärv av egnahemsfastighet i vissa fall","subtitel":"","status":"","text":"1 § Enligt denna förordning får lån (förvärvslån) lämnas för förvärv\r\nfrån staten av egnahemsfastighet som har\r\n\r\n1. inlösts enligt 56 a § arbetsmarknadskungörelsen (1966:368),\r\n\r\n2. avstyckats från jordbruksfastighet genom åtgärder i samband med\r\njordbrukets rationalisering.\r\n\r\nFörvärvslån får lämnas även för förvärv av egnahemsfastighet, som kan\r\nlösas in enligt arbetsmarknadskungörelsen, om inlösen undviks därigenom.\r\nFörordning (1985:458).\r\n\r\n2 § Låneverksamheten utövas av plan- och bostadsverket,\r\nlänsbostadsnämnderna och kommunerna.\r\n\r\nOrgan för kommuns låneförmedling och annan verksamhet som sammanhänger\r\nmed den (förmedlingsorgan) är kommunstyrelsen om ej kommunen har\r\nbeslutat att verksamheten skall utövas av annat kommunalt organ.\r\n\r\nI ärende enligt denna förordning, vari kommun ej är sökande, får beslut\r\nsom innebär avvikelse från förmedlingsorganets yttrande i ärendet\r\nmeddelas endast om särskilda skäl föreligger. Förordning (1988:303).\r\n\r\n3 § Förvärvslån får efter ansökan beviljas enskild person, kommun eller\r\nallmännyttigt bostadsföretag.\r\n\r\nÄr sökanden enskild person och överstiger köpeskillingen 10 000 kronor\r\nlämnas lån endast under förutsättning att kommunen har åtagit sig att,\r\nom förlust uppkommer på lånet, svara för sådan förlust intill ett belopp\r\nsom motsvarar 40 procent av låneskulden vid förlusttillfället.\r\nFörordning (1985:458).\r\n\r\n4 § Förvärvslån till kommun eller allmännyttigt bostadsföretag utgår\r\nhögst med belopp som motsvarar köpeskillingen, minskad med låneskuld för\r\nvilken panträtt har upplåtits i fastigheten och som övertages i samband\r\nmed förvärvet.\r\n\r\nTill enskild person utgår lån högst med belopp som motsvarar skillnaden\r\nmellan 95 procent av köpeskillingen och låneskuld som avses i första\r\nstycket.\r\n\r\nLån utgår ej om det belopp som beräknas enligt första eller andra\r\nstycket är obetydligt i förhållande till köpeskillingen.\r\n\r\n5 § I fråga om säkerhet för förvärvslån tillämpas bestämmelserna i 23 §\r\nnybyggnadslåneförordningen för bostäder (1986:692).\r\n\r\nUpplåtes till säkerhet för lånet panträtt i fastighet, skall\r\ninteckningen vara belägen inom 95 procent av köpeskillingen när\r\nlåntagaren är enskild person och inom 100 procent av köpeskillingen i\r\növriga fall. Förordning (1986:713).\r\n\r\n6 § Förvärvslån löper med bunden ränta från utbetalningsdagen.\r\nRäntesatsen samt tidpunkt och villkor för ändring av räntesatsen under\r\nlånetiden fastställs med ledning av räntevillkoren för obligationslån\r\nutgivna av Statens Bostadsfinansieringsaktiebolag. Förordning\r\n(1988:303).\r\n\r\n7 § Förvärvslån amorteras med lika stora årliga belopp under det antal\r\når som länsbostadsnämnden bestämmer med hänsyn till husets återstående\r\nanvändningstid och övriga omständigheter. Amorteringstiden får dock\r\nuppgå till högst 20 år.\r\n\r\nAmorteringstiden räknas från utbetalningsdagen. Förordning (1988:303).\r\n\r\n8 § Fråga om lån prövas av länsbostadsnämnden. Ansökan skall ställas\r\ntill länsbostadsnämnden och ges in till förmedlingsorganet som med eget\r\nyttrande skall sända handlingarna i ärendet till länsbostadsnämnden.\r\n\r\nÄr ansökningshandlingarna ofullständiga eller på annat sätt\r\nbristfälliga, skall förmedlingsorganet ge sökanden tillfälle att\r\navhjälpa bristen innan handlingarna översändes.\r\n\r\n9 § Bestämmelserna i 34 § andra och tredje styckena, 38 och 45 §§\r\nnybyggnadslåneförordningen för bostäder (1986:692) samt 2, 12, 16, 20\r\noch 21 §§, 22 § första stycket, 23 §, 23 a §, 29 § första stycket och 30\r\n§ förordningen (1986:694) om handläggning, förvaltning, m. m. av\r\nbostadslån och räntebidrag gäller i tillämpliga delar i fråga om\r\nförvärvslån.\r\n\r\nOm fastigheten övergår till en ny ägare som är enskild person och om\r\nköpeskillingen när lånet beviljades översteg 10 000 kronor, får lånet\r\növertas endast om kommunen åtar sig sådant ansvar som avses i 3 § andra\r\nstycket. Förordning (1989:862).\r\n\r\n10 § Talan mot länsbostadsnämndens beslut enligt denna förordning föres\r\nhos plan- och bostadsverket genom besvär.\r\n\r\nTalan mot plan- och bostadsverkets beslut föres hos regeringen genom\r\nbesvär. Förordning (1988:303).\r\n\r\n11 § Föreskrifter för verkställighet av denna förordning meddelas av\r\nplan- och bostadsverket. Förordning (1988:303).\r\n\r\nÖvergångsbestämmelser\r\n\r\n1985:458\r\n\r\nDenna förordning träder i kraft den 1 juli 1985. Förordningen gäller\r\näven i fråga om förvärvslån som har beviljats men inte betalats ut före\r\nikraftträdandet.\r\n\r\n1988:303\r\n\r\nDenna förordning träder i kraft den 1 juli 1988.\r\n\r\nBestämmelserna i 7 § i sin nya lydelse tillämpas även i ärenden\r\ni vilka förvärvslån har beviljats men inte betalats ut före\r\nikraftträdandet.\r\n\r\n1989:862\r\n\r\nDenna förordning träder i kraft den 1 januari 1990.\r\n\r\nÄldre bestämmelser tillämpas fortfarande om fastigheten övergått till ny\r\nägare före ikraftträdandet. I sådana ärenden skall dock de nya\r\nbestämmelserna tillämpas, om den nya ägaren begär det och frågan om\r\nersättningslån ännu inte prövats av länsbostadsnämnden.\r\n\r\n1991:1929\r\n\r\nDe upphävda förordningarna gäller fortfarande i fråga om lån eller\r\nbidrag som har beviljats enligt förordningarna.","html":"<style>\r\n.document div {\r\n    overflow: visible !important;\r\n    width: 550px !important;\r\n}\r\n</style>\r\n\r\n<b>SFS nr</b>:     \r\n1976:257<br />\r\n\r\n<b>Departement/myndighet</b>:   \r\nBostadsdepartementet <br />  \r\n\r\n<b>Utfärdad</b>:   \r\n1976-05-13 <br />  \r\n\r\n\r\n<b>Upphävd</b>: \r\n1992-01-01<br />\r\n\r\n<b>Författningen har upphävts genom</b>: \r\nSFS 1991:1929<br />\r\n\r\n\r\n<b>Ändringsregister</b>:   \r\n<a href=\"http://rkrattsbaser.gov.se/sfsr?bet=1976:257\">SFSR (Regeringskansliet)</a> <br />\r\n\r\n<b>Källa</b>: \r\n\r\n<a href=\"http://rkrattsbaser.gov.se/sfst?bet=1976:257\">Fulltext (Regeringskansliet)</a> \r\n\r\n\r\n<br />\r\n\r\n<hr />\r\n\r\n\r\n\r\n<style><!-- \r\ndiv.sfstoc {padding:10px;position:relative;width:90%;border-bottom:#ccc 1px solid;font-size:85%;} \r\n--></style>\r\n<div class=\"sfstoc\"><h3>Innehåll:</h3>\r\n<ul>\r\n <li><a href=\"#overgang\">Övergångsbestämmelser</a></li>\r\n</ul>\r\n</div>\r\n<div><p><a name=\"S1\"></a></p><a class=\"paragraf\" name=\"P1\"><b>1 §</b></a>   Enligt denna förordning får lån (förvärvslån) lämnas för förvärv från staten av egnahemsfastighet som har<br />\r\n   1. inlösts enligt 56 a § arbetsmarknadskungörelsen (1966:368),<br />\r\n   2. avstyckats från jordbruksfastighet genom åtgärder i samband med jordbrukets rationalisering.<p><a name=\"P1S2\"></a></p>\r\n\r\nFörvärvslån får lämnas även för förvärv av egnahemsfastighet, som kan lösas in enligt arbetsmarknadskungörelsen, om inlösen undviks därigenom.<br />\r\nFörordning (1985:458).<p><a name=\"P1S3\"></a></p><a class=\"paragraf\" name=\"P2\"><b>2 §</b></a>   Låneverksamheten utövas av plan- och bostadsverket, länsbostadsnämnderna och kommunerna.<p><a name=\"P2S2\"></a></p>\r\n\r\nOrgan för kommuns låneförmedling och annan verksamhet som sammanhänger med den (förmedlingsorgan) är kommunstyrelsen om ej kommunen har beslutat att verksamheten skall utövas av annat kommunalt organ.<p><a name=\"P2S3\"></a></p>\r\n\r\nI ärende enligt denna förordning, vari kommun ej är sökande, får beslut som innebär avvikelse från förmedlingsorganets yttrande i ärendet meddelas endast om särskilda skäl föreligger. Förordning (1988:303).<p><a name=\"P2S4\"></a></p><a class=\"paragraf\" name=\"P3\"><b>3 §</b></a>   Förvärvslån får efter ansökan beviljas enskild person, kommun eller allmännyttigt bostadsföretag.<p><a name=\"P3S2\"></a></p>\r\n\r\nÄr sökanden enskild person och överstiger köpeskillingen 10 000 kronor lämnas lån endast under förutsättning att kommunen har åtagit sig att, om förlust uppkommer på lånet, svara för sådan förlust intill ett belopp som motsvarar 40 procent av låneskulden vid förlusttillfället.<br />\r\nFörordning (1985:458).<p><a name=\"P3S3\"></a></p><a class=\"paragraf\" name=\"P4\"><b>4 §</b></a>   Förvärvslån till kommun eller allmännyttigt bostadsföretag utgår högst med belopp som motsvarar köpeskillingen, minskad med låneskuld för vilken panträtt har upplåtits i fastigheten och som övertages i samband med förvärvet.<p><a name=\"P4S2\"></a></p>\r\n\r\nTill enskild person utgår lån högst med belopp som motsvarar skillnaden mellan 95 procent av köpeskillingen och låneskuld som avses i första stycket.<p><a name=\"P4S3\"></a></p>\r\n\r\nLån utgår ej om det belopp som beräknas enligt första eller andra stycket är obetydligt i förhållande till köpeskillingen.<p><a name=\"P4S4\"></a></p><a class=\"paragraf\" name=\"P5\"><b>5 §</b></a>   I fråga om säkerhet för förvärvslån tillämpas bestämmelserna i 23 §\r\nnybyggnadslåneförordningen för bostäder (1986:692).<p><a name=\"P5S2\"></a></p>\r\n\r\nUpplåtes till säkerhet för lånet panträtt i fastighet, skall inteckningen vara belägen inom 95 procent av köpeskillingen när låntagaren är enskild person och inom 100 procent av köpeskillingen i övriga fall. Förordning (1986:713).<p><a name=\"P5S3\"></a></p><a class=\"paragraf\" name=\"P6\"><b>6 §</b></a>   Förvärvslån löper med bunden ränta från utbetalningsdagen.<br />\r\nRäntesatsen samt tidpunkt och villkor för ändring av räntesatsen under lånetiden fastställs med ledning av räntevillkoren för obligationslån utgivna av Statens Bostadsfinansieringsaktiebolag. Förordning\r\n(1988:303).<p><a name=\"P6S2\"></a></p><a class=\"paragraf\" name=\"P7\"><b>7 §</b></a>   Förvärvslån amorteras med lika stora årliga belopp under det antal år som länsbostadsnämnden bestämmer med hänsyn till husets återstående användningstid och övriga omständigheter. Amorteringstiden får dock uppgå till högst 20 år.<p><a name=\"P7S2\"></a></p>\r\n\r\nAmorteringstiden räknas från utbetalningsdagen. Förordning (1988:303).<p><a name=\"P7S3\"></a></p><a class=\"paragraf\" name=\"P8\"><b>8 §</b></a>   Fråga om lån prövas av länsbostadsnämnden. Ansökan skall ställas till länsbostadsnämnden och ges in till förmedlingsorganet som med eget yttrande skall sända handlingarna i ärendet till länsbostadsnämnden.<p><a name=\"P8S2\"></a></p>\r\n\r\nÄr ansökningshandlingarna ofullständiga eller på annat sätt bristfälliga, skall förmedlingsorganet ge sökanden tillfälle att avhjälpa bristen innan handlingarna översändes.<p><a name=\"P8S3\"></a></p><a class=\"paragraf\" name=\"P9\"><b>9 §</b></a>   Bestämmelserna i 34 § andra och tredje styckena, 38 och 45 §§\r\nnybyggnadslåneförordningen för bostäder (1986:692) samt 2, 12, 16, 20 och 21 §§, 22 § första stycket, 23 §, 23 a §, 29 § första stycket och 30\r\n§ förordningen (1986:694) om handläggning, förvaltning, m. m. av bostadslån och räntebidrag gäller i tillämpliga delar i fråga om förvärvslån.<p><a name=\"P9S2\"></a></p>\r\n\r\nOm fastigheten övergår till en ny ägare som är enskild person och om köpeskillingen när lånet beviljades översteg 10 000 kronor, får lånet övertas endast om kommunen åtar sig sådant ansvar som avses i 3 § andra stycket. Förordning (1989:862).<p><a name=\"P9S3\"></a></p><a class=\"paragraf\" name=\"P10\"><b>10 §</b></a>   Talan mot länsbostadsnämndens beslut enligt denna förordning föres hos plan- och bostadsverket genom besvär.<p><a name=\"P10S2\"></a></p>\r\n\r\nTalan mot plan- och bostadsverkets beslut föres hos regeringen genom besvär. Förordning (1988:303).<p><a name=\"P10S3\"></a></p><a class=\"paragraf\" name=\"P11\"><b>11 §</b></a>   Föreskrifter för verkställighet av denna förordning meddelas av plan- och bostadsverket. Förordning (1988:303).<p><a name=\"P11S2\"></a></p><br /><h3 name=\"overgang\"><a name=\"overgang\">Övergångsbestämmelser</a></h3>\r\n1985:458<p><a name=\"P11S2\"></a></p>\r\n\r\nDenna förordning träder i kraft den 1 juli 1985. Förordningen gäller även i fråga om förvärvslån som har beviljats men inte betalats ut före ikraftträdandet.<p><a name=\"P11S3\"></a></p>\r\n\r\n1988:303<p><a name=\"P11S4\"></a></p>\r\n\r\nDenna förordning träder i kraft den 1 juli 1988.<p><a name=\"P11S5\"></a></p>\r\n\r\nBestämmelserna i 7 § i sin nya lydelse tillämpas även i ärenden i vilka förvärvslån har beviljats men inte betalats ut före ikraftträdandet.<p><a name=\"P11S6\"></a></p>\r\n\r\n1989:862<p><a name=\"P11S7\"></a></p>\r\n\r\nDenna förordning träder i kraft den 1 januari 1990.<p><a name=\"P11S8\"></a></p>\r\n\r\nÄldre bestämmelser tillämpas fortfarande om fastigheten övergått till ny ägare före ikraftträdandet. I sådana ärenden skall dock de nya bestämmelserna tillämpas, om den nya ägaren begär det och frågan om ersättningslån ännu inte prövats av länsbostadsnämnden.<p><a name=\"P11S9\"></a></p>\r\n\r\n1991:1929<p><a name=\"P11S10\"></a></p>\r\n\r\nDe upphävda förordningarna gäller fortfarande i fråga om lån eller bidrag som har beviljats enligt förordningarna.<p><a name=\"P11S11\"></a></p></div>"},"dokuppgift":{"uppgift":[{"kod":"artal","namn":"artal","text":"1976"},{"kod":"utfardad","namn":"utfardad","text":"1976-05-13"},{"kod":"upphnr","namn":"upphnr","text":"SFS 1991:1929"},{"kod":"upphavd","namn":"upphavd","text":"1992-01-01 00:00:00"},{"kod":"utdrag","namn":"utdrag","text":"1 § Enligt denna förordning får lån (förvärvslån) lämnas för förvärv\r\nfrån staten av egnahemsfastighet som har\r\n\r\n1. inlösts enligt 56 a § arbetsmarknadskungörelsen (1966:368),\r\n\r\n2. avstyckats från jordbruksfastighet genom åtgärder i samband med\r\njordbrukets"}]}}}