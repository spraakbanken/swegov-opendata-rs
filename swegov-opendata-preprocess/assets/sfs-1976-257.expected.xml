<file xmlns="">
    <dokument xmlns="" dok_id="sfs-1976-257"
    >
        <text datatyp="huvuddokument" hangar_id="" rm="1976" beteckning="1976:257" typ="sfs"
            subtyp="sfst" dokumentnamn="Svensk författningssamling"
            organ="Bostadsdepartementet" nummer="257" slutnummer="0" datum="1976-05-13 00:00:00"
            publicerad="2016-02-11 15:25:48" systemdatum="2016-02-11 15:25:48"
            titel="Förordning (1976:257) om lån för förvärv av egnahemsfastighet i vissa fall"
            status="" upphavd="1992-01-01" upphnr="SFS 1991:1929"
            ändringsregister="http://rkrattsbaser.gov.se/sfsr?bet=1976:257"
            källa="http://rkrattsbaser.gov.se/sfst?bet=1976:257">
            <p> 1 § Enligt denna förordning får lån (förvärvslån) lämnas för förvärv från staten av
                egnahemsfastighet som har<br /> 1. inlösts enligt 56 a § arbetsmarknadskungörelsen
                (1966:368),<br /> 2. avstyckats från jordbruksfastighet genom åtgärder i samband med
                jordbrukets rationalisering.</p>
            <p>Förvärvslån får lämnas även för förvärv av egnahemsfastighet, som kan lösas in enligt
                arbetsmarknadskungörelsen, om inlösen undviks därigenom.<br />Förordning (1985:458).</p>
            <p>2 § Låneverksamheten utövas av plan- och
                bostadsverket,
                länsbostadsnämnderna och kommunerna.</p>
            <p>Organ för kommuns låneförmedling och annan
                verksamhet som sammanhänger med den (förmedlingsorgan) är kommunstyrelsen om ej
                kommunen
                har
                beslutat att verksamheten skall utövas av annat kommunalt organ.</p>
            <p>I ärende enligt denna
                förordning, vari kommun ej
                är sökande, får beslut som innebär avvikelse från förmedlingsorganets yttrande i
                ärendet
                meddelas endast om särskilda skäl föreligger. Förordning (1988:303).</p>
            <p>3 § Förvärvslån
                får efter ansökan beviljas enskild person, kommun
                eller allmännyttigt bostadsföretag.</p>
            <p>Är sökanden enskild person och överstiger köpeskillingen 10 000 kronor lämnas lån
                endast under förutsättning att kommunen har åtagit sig att, om förlust uppkommer på
                lånet, svara för sådan förlust intill ett belopp som motsvarar 40 procent av
                låneskulden vid förlusttillfället.<br />Förordning (1985:458).</p>
            <p>4
                § Förvärvslån till kommun eller allmännyttigt bostadsföretag
                utgår högst med belopp som motsvarar köpeskillingen, minskad med låneskuld för
                vilken
                panträtt har upplåtits i fastigheten och som övertages i samband med förvärvet.</p>
            <p>Till
                enskild person utgår lån högst med belopp
                som motsvarar skillnaden mellan 95 procent av köpeskillingen och låneskuld som avses
                i
                första stycket.</p>
            <p>Lån utgår ej om det belopp som beräknas enligt
                första eller andra stycket är obetydligt i förhållande till köpeskillingen.</p>
            <p>5 § I
                fråga om säkerhet för förvärvslån tillämpas bestämmelserna i
                23 § nybyggnadslåneförordningen för bostäder (1986:692).</p>
            <p>Upplåtes till säkerhet
                för lånet panträtt i
                fastighet, skall inteckningen vara belägen inom 95 procent av köpeskillingen när
                låntagaren
                är enskild person och inom 100 procent av köpeskillingen i övriga fall. Förordning
                (1986:713).</p>
            <p>6 § Förvärvslån löper med bunden ränta från utbetalningsdagen.<br />Räntesatsen samt
                tidpunkt och villkor för ändring av räntesatsen under lånetiden fastställs med
                ledning av räntevillkoren för obligationslån utgivna av Statens
                Bostadsfinansieringsaktiebolag. Förordning (1988:303).</p>
            <p>7 § Förvärvslån amorteras med lika stora årliga belopp under
                det
                antal år som länsbostadsnämnden bestämmer med hänsyn till husets återstående
                användningstid
                och övriga omständigheter. Amorteringstiden får dock uppgå till högst 20 år.</p>
            <p>Amorteringstiden
                räknas från utbetalningsdagen.
                Förordning (1988:303).</p>
            <p>8 § Fråga om lån prövas av länsbostadsnämnden. Ansökan
                skall
                ställas till länsbostadsnämnden och ges in till förmedlingsorganet som med eget
                yttrande
                skall sända handlingarna i ärendet till länsbostadsnämnden.</p>
            <p>Är ansökningshandlingarna
                ofullständiga eller på
                annat sätt bristfälliga, skall förmedlingsorganet ge sökanden tillfälle att avhjälpa
                bristen
                innan handlingarna översändes.</p>
            <p>9 § Bestämmelserna i 34 § andra och tredje
                styckena, 38 och 45
                §§ nybyggnadslåneförordningen för bostäder (1986:692) samt 2, 12, 16, 20 och 21 §§,
                22 §
                första stycket, 23 §, 23 a §, 29 § första stycket och 30 § förordningen (1986:694)
                om
                handläggning, förvaltning, m. m. av bostadslån och räntebidrag gäller i tillämpliga
                delar i
                fråga om förvärvslån.</p>
            <p>Om fastigheten övergår till en ny ägare som är
                enskild person och om köpeskillingen när lånet beviljades översteg 10 000 kronor,
                får
                lånet
                övertas endast om kommunen åtar sig sådant ansvar som avses i 3 § andra stycket.
                Förordning
                (1989:862).</p>
            <p>10 § Talan mot länsbostadsnämndens beslut enligt denna förordning
                föres hos plan- och bostadsverket genom besvär.</p>
            <p>Talan mot plan- och bostadsverkets
                beslut föres
                hos regeringen genom besvär. Förordning (1988:303).</p>
            <p>11 § Föreskrifter för
                verkställighet av denna förordning meddelas
                av plan- och bostadsverket. Förordning (1988:303).</p>
            <p>Övergångsbestämmelser</p>
            <p>1985:458</p>

            <p> Denna förordning träder i kraft den 1 juli 1985.
                Förordningen gäller även i fråga om förvärvslån som har beviljats men inte betalats
                ut
                före
                ikraftträdandet.</p>
            <p>1988:303</p>
            <p>Denna förordning träder i kraft den 1 juli 1988.
            </p>
            <p>
                Bestämmelserna i 7 § i sin nya lydelse tillämpas även i ärenden i vilka
                förvärvslån har beviljats men inte betalats ut före ikraftträdandet.</p>
            <p>1989:862</p>
            <p>Denna förordning träder i kraft den 1 januari 1990.</p>
            <p>Äldre bestämmelser tillämpas fortfarande om fastigheten övergått till ny ägare
                före ikraftträdandet. I sådana ärenden skall dock de nya bestämmelserna tillämpas,
                om
                den
                nya ägaren begär det och frågan om ersättningslån ännu inte prövats av
                länsbostadsnämnden.</p>
            <p>1991:1929</p>
            <p>De upphävda förordningarna gäller fortfarande i fråga om lån eller bidrag som
                har beviljats enligt förordningarna.</p>
        </text>
    </dokument>

</file>